### DISCUSSION TITLE
Is it possible to publish as tar.gz and not as zip?

### DISCUSSION THREAD

**Discussion by VictorDC92** [Elendev/nexus-composer-push#67] [General]
I want to publish my files as tar.gz and not as a zip. It's possible?

**Comment by Elendev** [comment]
Hello @VictorDC92 currently there is only a ZIP archiver (https://github.com/Elendev/composer-push/blob/master/src/ZipArchiver.php).
But it's a good point, and the plugin should allow to have multiple archive format, using an argument when running the command:
`--archive-format=tar.gz` for example.
Feel free to contribute if you want to propose a PR 👍
